shap/explainers/_tree.py

```diff
@@ -348,8 +360,11 @@ def shap_values(self, X, y=None, tree_limit=None, approximate=False, check_addit
                         approx_contribs=approximate, validate_features=False
                     )
                 except ValueError as e:
-                        raise ValueError("This reshape error is often caused by passing a bad data matrix to SHAP. " \
-                                         "See https://github.com/slundberg/shap/issues/580") from e
+                    emsg = (
+                        "This reshape error is often caused by passing a bad data matrix to SHAP. "
+                        "See https://github.com/slundberg/shap/issues/580."
+                    )
+                    raise ValueError(emsg) from e
```

Comment:
What is the purpose of this refactor? IMO the original is clearer: directly putting the string in the call to `ValueError`

Reply:
(Resolved in other thread)